package.json

```diff
@@ -10,7 +10,16 @@
     "test": "react-scripts test --watchAll=false",
     "test:coverage": "react-scripts test --coverage --watchAll=false",
     "test:update": "react-scripts test --updateSnapshot --watchAll=false",
-    "coveralls": "react-scripts test --coverage --watchAll=false && cat ./coverage/lcov.info | coveralls"
+    "coveralls": "react-scripts test --coverage --watchAll=false && cat ./coverage/lcov.info | coveralls",
+    "storybook": "start-storybook",
+    "build-storybook": "build-storybook -c .storybook -o .out"
```

Comment:
You should export document in docs folder, because github page will look into docs folder for deploy document[ref](https://help.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#choosing-a-publishing-source)